src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useThorStreamingProgress.tsx

```diff
@@ -56,6 +56,7 @@ const getStreamingSwapMetadata = (
 }
 
 export const useThorStreamingProgress = (
+  _tradeQuoteStep: TradeQuoteStep,
```

Comment:
rm me, this is an addition and unused

Reply:
https://github.com/shapeshift/web/pull/8152/commits/3e1a7b69409db7f52c03242d38acbafbb466ea64

Reply:
thanks for refactoring this, I added it because the signature changed on CF side to get the TradeQuoteStep in to get access to its data.